sa3ed_app/www/lost_person_search/index.html

```diff
@@ -1,4 +1,10 @@
 {% extends base_template_path %}
 {% block content %}
-<div id="div_test_response"></div>
+<link rel="stylesheet" href="index.css" />
```

Comment:
if you are using naming like: index.html & index.css this sheet will be auto applied (no need for the link). this should also be applied in other files

Reply:
This issue should be solved by Commit 751b36a